packages/ckeditor5-block-quote/docs/features/block-quote.md

```diff
@@ -50,9 +61,9 @@ ClassicEditor
 
 ## Disallowing nesting block quotes
```

Comment:
I would add a "Configuration" section lvl 2 here, and then make "Disallow nesting block quotes" as its subsection lvl 3.

Reply:
Addressed in commit.